code/_onclick/item_attack.dm

```diff
@@ -10,7 +10,7 @@
 /obj/item/proc/melee_attack_chain(mob/user, atom/target, params)
 	var/is_right_clicking = LAZYACCESS(params2list(params), RIGHT_CLICK)
 
-	if(tool_behaviour && (target.tool_act(user, src, tool_behaviour, is_right_clicking) & TOOL_ACT_MELEE_CHAIN_BLOCKING))
+	if(tool_behaviour && target.tool_act(user, src, tool_behaviour, is_right_clicking))
```

Comment:
This was done to correct unintended behaviour, returning any truthy value from tool_act is supposed to short circuit the melee chain

Reply:
This does not appear to be unintended. According to the comment for TOOL_ACT_MELEE_CHAIN_BLOCKING, this appears to be the desired behavior.

Reply:
```suggestion
	if(tool_behaviour && (target.tool_act(user, src, tool_behaviour, is_right_clicking) & TOOL_ACT_MELEE_CHAIN_BLOCKING))
```